src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/SemanticModelExtensions.cs

```diff
@@ -90,15 +91,6 @@ public static ISymbol GetEnclosingNamedTypeOrAssembly(this SemanticModel semanti
     public static INamespaceSymbol? GetEnclosingNamespace(this SemanticModel semanticModel, int position, CancellationToken cancellationToken)
         => semanticModel.GetEnclosingSymbol<INamespaceSymbol>(position, cancellationToken);
 
-    public static IEnumerable<ISymbol> GetExistingSymbols(
-        this SemanticModel semanticModel, SyntaxNode? container, CancellationToken cancellationToken, Func<SyntaxNode, bool>? descendInto = null)
-    {
-        // Ignore an anonymous type property or tuple field.  It's ok if they have a name that
-        // matches the name of the local we're introducing.
-        return semanticModel.GetAllDeclaredSymbols(container, cancellationToken, descendInto)
-            .Where(s => !s.IsAnonymousTypeProperty() && !s.IsTupleField());
```

Comment:
i moved this filtering into GetAllDeclaredSymbols as all callers want this behavior.